hackchicago/hackchicago
# Dynamically update text snippets by time

Replaces #12.

## To test:

- Signed up (works across all)
  - Run the following in your console:
    ```
    Cookies.set('hasSignedUp','true');window.location.reload();
    ```
- Before event
  - Open the page as normal, everything should work properly
- After signups close
  - Run the following in your console:
    ```
    signupEnd=moment('2017-07-14 23:59');checkTime();
    ```
- After event
  - Run the following in your console:
    ```
    signupEnd=moment('2017-07-14 23:59');hackChicagoEnd=moment('2017-07-22 12:00');checkTime();
    ```

## Screenshots:
- Before event
    ![image](https://user-images.githubusercontent.com/16865690/40460547-266243fc-5ebc-11e8-860d-d99c7df31012.png)
- After signups close
    ![image](https://user-images.githubusercontent.com/16865690/40460573-4bad46de-5ebc-11e8-8e65-602ac481eaf8.png)
- After event
    ![image](https://user-images.githubusercontent.com/16865690/40460644-99d45faa-5ebc-11e8-9ac3-ffd9d9a67c95.png)

## Issues

Please let me know if you encounter any issues and provide a screenshot.

#### Let's merge ASAP.
